pyproject.toml

```diff
@@ -14,9 +14,11 @@ requires-python = "==3.8.*"
 dependencies = [
     "tqdm",
     "numpy>=1.21",
-    "datasets==2.*",
     "brainscore_core@git+https://github.com/brain-score/core.git@main",
-    "brainio@git+https://github.com/brain-score/brainio.git@main"
+    "brainio@git+https://github.com/brain-score/brainio.git@main",
```

Comment:
are there any plans to release-control any of these repos? currently our specification is the `main` branch.
it would be nice if they were on `pypi` too

Reply:
yes, I would love that! Can you help make that happen?

Reply:
#44